waiter/config-full.edn

```diff
@@ -382,7 +382,9 @@
                                              :kubernetes {;; a factory function must be provided
                                                           ;; see the :kubernetes entry for other configuration options
                                                           :factory-fn waiter.scheduler.kubernetes/kubernetes-scheduler
-                                                          :todo :additional-config}}
+                                                          :todo :additional-config
+                                                          :proxy-options {:reverse-proxy-flag "GRPC_TRANSCODER"
+                                                                          :reverse-proxy-offset 1}}}
```

Comment:
This entry can be removed, the comment above is referring the user to the segment below.

Reply:
Done